widget/ui/src/components/FullExpandedQuote/FullExpandedQuote.tsx

```diff
@@ -168,7 +168,8 @@ export function FullExpandedQuote(props: PropTypes) {
               </Tooltip>
             </div>
             <StyledPriceImpact
-              size="xsmall"
+              size="small"
+              variant="full-expanded"
```

Comment:
`PriceImpact` shouldn't be related to `variants` and such props. For example, you can pass just `outputColor`.